README.md

```diff
@@ -42,9 +42,49 @@ You should use one of these three ways to import the DroneCore-Swift SDK but not
 
 ### Carthage
 
-Coming soon
+### Use Carthage to get the framework
 
-### Frameworks
+To use the DroneCore-Swift SDK in your iOS application, you can pull in this framework using [Carthage](https://github.com/Carthage/Carthage).
+
+To install carthage, it's easiest to use [Homebrew](https://brew.sh/):
+
+```
+brew install carthage
+```
+
+Then to add the framework, create the file `Cartfile` in your app's repository with below's content:
+
+```
+# Require the iOS framework of DroneCore SDK
+github "dronecore/DroneCore-Swift"
```

Comment:
So this just gets latest master or the latest semver tag?

Reply:
@JonasVautherin you probably need to add a tag here and then create a git tag (or use a branch for now).

Reply:
it get the latest version of master I think , but anyway we should add tag